src/core_plugins/kibana/public/management/sections/settings/index.html

```diff
@@ -21,10 +21,10 @@
       </div>
     </div>
 
-    <!-- Search -->
-    <form
-      role="form"
-      class="kuiVerticalRhythm"
+    <!-- Search and Import/Export -->
+    <div
```

Comment:
I love these improvements you're making man! I just have on request: can we wrap both the ToolBar and Table components inside of a ControlledTable? Like this:

```html
<div class="kuiControlledTable">
  <div class="kuiToolBar">etc</div>
  <div class="kuiTable">etc</div>
</div>
```

This will close the gap between the ToolBar and the Table, and you'll be able to remove `kuiVerticalRhythm` from both of them.